### ISSUE TITLE
BUG: Not actually crossing the line sends you to the future

### ISSUE THREAD

**Issue by TheAngryRaven** [TheAngryRaven/DovesLapTimer#7]
# Problem
If for some reason you enter a line's threshold, but do not cross the line. The system absolutely freaks out.

## Description
I'm not entirely sure i can think of a scenario where this would happen in the real world

## Proposed Solution
1. Monitor current driver line side
2. If we exit the threshold but never cross the line, do not actually calculate crossing!

## Concerns
1. Low priority
2. High Complexity to solve

## Blocked By: #4

**Comment by TheAngryRaven** [comment]
Possible fixed with latest release while trying to fixed #4 which will be placed on hold for now.
